scenes/game_elements/characters/enemies/guard/components/guard.gd

```diff
@@ -409,6 +417,38 @@ func edit_patrol_path() -> void:
 		get_parent().add_child(patrol_path)
 		patrol_path.owner = owner
 		patrol_path.global_position = global_position
-		patrol_path.curve = Curve2D.new()
+		var patrol_path_curve: Curve2D = Curve2D.new()
+		patrol_path.curve = patrol_path_curve
 		patrol_path.name = "%s-PatrolPath" % name
+		patrol_path_curve.add_point(Vector2.ZERO)
+		patrol_path_curve.add_point(Vector2.UP.rotated(randf_range(0, PI * 2.0)) * 150.0)
```

Comment:
I'm not really sure about the second point being random. I think a horizontal line or a vertical line will be more predictable. But maybe I'm wrong and this is fun!

Reply:
Mmmm, yeah, let's keep it simple and just make a fixed horizontal path and have the students modify it.